src/ttmask/_cli.py

```diff
@@ -1,3 +1,3 @@
 import typer
 
-cli = typer.Typer(name="ttmask")
+cli = typer.Typer(name="ttmask")
```

Comment:
github likes having a newline at the end of files, auto format in pycharm normally does this